packages/shared_preferences/shared_preferences/extension/devtools/.pubignore

```diff
@@ -0,0 +1 @@
+!build
```

Comment:
The `extension/devtools/build` folder will need to either be checked in, or the `devtools_extensions build_and_copy` command to generate the extension assets from the source code will need to be included as part of the pub publish flow for `shared_preferences`. The latter would be more safe since the machine doing the publish would also be the one to build the extension assets. However, the extension likely will not need to be regenerated for each publish of `shared_preferences` - only if there are updates to the extension. So that should be taken into consideration. 

@stuartmorgan do you have a preference or some guidance on this?

Reply:
> The `extension/devtools/build` folder will need to either be checked in, or the `devtools_extensions build_and_copy` command to generate the extension assets from the source code will need to be included as part of the pub publish flow for `shared_preferences`. The latter would be more safe since the machine doing the publish would also be the one to build the extension assets.

We definitely don't want to introduce new processes that involve individuals building precompiled binaries on uncontrolled machines and then checking them in.

> However, the extension likely will not need to be regenerated for each publish of `shared_preferences` - only if there are updates to the extension. So that should be taken into consideration.

How long does that build take?